common/main/java/com/couchbase/lite/AbstractReplicator.java

```diff
@@ -382,6 +382,15 @@ public void removeChangeListener(@NonNull ListenerToken token) {
         }
     }
 
+    // When a replicator is closed it is detached from its from LiteCore and receives no further status updates.
```

Comment:
As close() is the public API method, mention about LiteCore could be confusing to users. Maybe move the internal comment inside the method itself?

Also calling close() while the replicator is not stopped, seems to be dangerous and could cause unexpectedly crash I guessed. I think it might be worth mentioning about that as well.

Reply:
Good points:
- That comment is not a javadoc comment.  It is visible only to people who are reading the code...  like you and me ;-P
- This is not the right solution.  I have a ticket to fix it for real.  None the less, I think it would be hard to cause a crash....  Maybe if you close while a push filter is running...

The final solution may be to disallow closing *unless* the replicator is stopped...